content-services/7.3/install/ansible.md

```diff
@@ -136,7 +136,9 @@ The table below shows the version of the components deployed by the playbook for
 
 Not all distributions of Linux may match the version requirements for Ansible and its dependencies. Below we describe how to configure a control node with one of the many ways to set a Python virtual environment. With Python `virtualenvs` you can install the exact same versions of Ansible we use when testing without impacting your system installation of Python. By doing so, you're ensuring the best chances of success.
 
-1. Download the Ansible playbook [zip file](https://nexus.alfresco.com/nexus/service/local/repositories/releases/content/org/alfresco/alfresco-ansible-deployment/2.2.1/alfresco-ansible-deployment-2.2.1.zip){:target="_blank"}.
+1. Download the Ansible playbook: `alfresco-ansible-deployment-x.x`, where `x.x` is the version number.
+
+   You can download the Ansible playbook from the Alfresco's Nexus repository available at https://nexus.alfresco.com/nexus. Find the zip file in the repository by searching for `alfresco-ansible-deployment`. For the supported version of the playbook, see [Alfresco Ansible Deployment](https://alfresco.github.io/alfresco-ansible-deployment/overview.html){:target="_blank"}.

```

Comment:
We can also mention way to get ansible deployment from https://github.com/Alfresco/alfresco-ansible-deployment/releases